json/wavelog.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "Wavelog",
+    "slug": "wavelog",
+    "categories": [
+        0
+    ],
+    "date_created": "2024-11-12",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "80",
+    "documentation": "https://github.com/wavelog/wavelog/wiki",
+    "website": "https://www.wavelog.org/",
+    "logo": "https://raw.githubusercontent.com/wavelog/wavelog/refs/heads/master/assets/logo/wavelog_logo_cosmo.png",
+    "description": "Wavelog is a self-hosted PHP application that allows you to log your amateur radio contacts anywhere. All you need is a web browser and active internet connection.",
+    "install_methods": [
+        {
+            "type": "default",
+            "script": "ct/wavelog.sh",
+            "resources": {
+                "cpu": "1",
+                "ram": "512",
+                "hdd": "2",
+                "os": "debian",
+                "version": "12"
+            }
+        }
+    ],
+    "default_credentials": {
+        "username": null,
+        "password": null
+    },
+    "notes": [
+        {
+            "text": "Database credentials are saved in: `~/wavelog.creds`",
```

```suggestion
            "text": "Database credentials: `cat wavelog.creds`",
```
Consistent with other scripts